wagtail/core/blocks/field_block.py

```diff
@@ -386,20 +391,22 @@ def __init__(self, choices=None, default=None, required=True, help_text=None, va
             self._constructor_kwargs['required'] = required
         if help_text is not None:
             self._constructor_kwargs['help_text'] = help_text
+        if widget is not None:
+            self._constructor_kwargs['widget'] = widget
```

Comment:
As per https://github.com/wagtail/wagtail/pull/5592#issuecomment-558861585 + subsequent discussion I believe we want to leave this out of `_constructor_kwargs`, so that we don't end up including it in frozen model definitions in migrations (where it will fail because Widget objects aren't serializable).